src/sqlancer/doris/DorisErrors.java

```diff
@@ -0,0 +1,120 @@
+package sqlancer.doris;
+
+import sqlancer.common.query.ExpectedErrors;
+
+public final class DorisErrors {
+
+    private DorisErrors() {
+    }
+
+    public static void addExpressionErrors(ExpectedErrors errors) {
```

Comment:
Many of the errors in this class are still from the DuckDB implementation. They likely do not apply to Doris. Could you please remove all the DuckDB ones?

Reply:
Thanks for your review, I have deleted the old error messages